src/components/CheckBox/CheckBox.tsx

```diff
@@ -1,6 +1,16 @@
-import { CheckIcon, TickIcon } from '@aftershootco/unicorn-icons'
+import { TickIcon } from '@aftershootco/unicorn-icons'
 import clsx from 'clsx'
 import React, { useCallback } from 'react'
+import { cn } from '../../utils/cn'
```

Comment:
@booi-dev 
create `index.ts` file and `import { cn } from '../../utils'` in utils instead of `cn.ts`.
That can include multiple utilities.